src/source/Rtp/RtpPacket.h

```diff
@@ -101,6 +101,8 @@ STATUS createBytesFromRtpPacket(PRtpPacket, PBYTE*, PUINT32);
 STATUS setBytesFromRtpPacket(PRtpPacket, PBYTE, UINT32);
 STATUS constructRtpPackets(PPayloadArray, UINT8, UINT16, UINT32, UINT32, PRtpPacket, UINT32);
 
+#define GET_UINT16_SEQ_NUM(seqIndex) ((UINT16) ((seqIndex) % (MAX_UINT16 + 1)))
```

Comment:
careful with MAX_UINT6 + 1. Need to check the standard but this might overflow if the compiler doesn't auto upgrade the type if seqIndex just happed to be UINT16. Better to use explicit cast like (UINT32)MAX_UINT16 + 1

Reply:
MAX_UINT16 is defined as 65535 not as (UINT16) 65535, so if it is running on 32/64 bit device, that MAX_UINT16+1 should be good enough.

Reply:
Actually, MAX_INT + 1 should be promoted to INT32 by the compiler, this should be OK